src/pages/contact-data-service-provider/[slug]/index.tsx

```diff
@@ -73,7 +73,7 @@ export default function ContactDataServiceProvider({ name, query }: ContactDataS
             >
               <ErrorSummary errors={errors} />
               <HoneyPot {...register('workEmailAddress')} />
-              <Fieldset>
+              <div>
```

Comment:
Removing the `fieldset` altogether is likely to raise a new accessibility issue. Could we instead add a `legend` providing the additional context to the form but have it visually hidden via the `govuk-visually-hidden` className ?

Reply:
I don't believe it's the right practice for the whole contact form to be wrapped within a fieldset and legend.
In FRF we have one example where we don't do this: 
https://findrecruitandfollowup.nihr.ac.uk/contact-research-support

GOVUK and NHS website don't do this either:
https://www.gov.uk/contact/govuk
https://www.nhs.uk/contact-us/nhs-app-contact-us/tell-us-what-you-need-help-with

Maybe we can briefly discuss on Thursdays call if we can fit it in

Reply:
Good point, that makes sense given these forms have no need for grouping 👍